src/Abp.Web.Common/Web/Settings/SettingScriptManager.cs

```diff
@@ -30,11 +40,28 @@ public SettingScriptManager(ISettingDefinitionManager settingDefinitionManager,
 
             var settingDefinitions = _settingDefinitionManager
                 .GetAllSettingDefinitions()
-                .Where(sd => sd.IsVisibleToClients);
+                .Where(sd => sd.ClientVisibility.IsVisible);
```

Comment:
What if ClientVisibility is null?

Reply:
It shouldn't be null becuase of `ClientVisibility = clientVisibility ?? new SettingDefinitionClientVisibility();` in the contstructor of SettingDefinition.cs.